src/libAtomVM/term.c

```diff
@@ -680,6 +680,49 @@ TermCompareResult term_compare(term t, term other, TermCompareOpts opts, GlobalC
     return result;
 }
 
+void term_get_function_mfa(term fun, Context *ctx, term *m, term *f, term *a)
```

Comment:
We either keep `Context *ctx` as first parameter, or last one, according if it is the "main focus" of the function or not, I would move it as last parameter. I'm not sure I did explain well this last point.

Here `Context *ctx` should be the last parameter.

Also as an additional feedback, I suggest avoiding using `Context` in `term.h`, since that would add an unnecessary coupling with `Context`, while we likely just need having a `GlobalContext` and a `Heap`.

Reply:
Thanks, that's good to know. We now only need global for creating atoms (function names).